app/uk/gov/hmrc/pdsauthcheckerstub/controllers/ValidateCustomsAuthController.scala

```diff
@@ -17,14 +17,18 @@
 package uk.gov.hmrc.pdsauthcheckerstub.controllers
 
 
+import play.api.libs.json.Json
 import play.api.mvc.{Action, ControllerComponents}
 import uk.gov.hmrc.pdsauthcheckerstub.models.PdsAuthRequest
+import uk.gov.hmrc.pdsauthcheckerstub.services.ValidateCustomsAuthService
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
 import javax.inject.{Inject, Singleton}
 @Singleton()
-class ValidateCustomsAuthController @Inject()(cc: ControllerComponents) extends BackendController(cc) {
+class ValidateCustomsAuthController @Inject()(cc: ControllerComponents, validateCustomsAuthService: ValidateCustomsAuthService) extends BackendController(cc) {
```

Comment:
ValidateCustomsAuthService seems an odd service for a stub. Surely this should be some sort of test data generator?